k8s/examples/wordpress/wordpress/20-statefulset.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: wordpress
+spec:
+  selector:
+    matchLabels:
+      app: wordpress
+      tier: frontend
+      storageos.com/replicas: "1"
+  serviceName: wordpress
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: wordpress
+        tier: frontend
+        storageos.com/replicas: "1"
+    spec:
+      serviceAccountName: wordpress
+      containers:
+      - name: wordpress
+        image: wordpress:5.2-apache
+        env:
+        - name: WORDPRESS_DB_HOST
+          value: wordpress-mysql
+        - name: WORDPRESS_DB_PASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: wordpress
+              key: wordpress-db-password
+        ports:
+        - name: wordpress
+          containerPort: 80
+        volumeMounts:
+        - name: wordpress-data
+          mountPath: /var/www/html
```

Comment:
We should look into saving only the files that need to be shared across instances. Otherwise we are putting unnecessary load to the nfs share. Which is already the less performant component of the setup. I can help with setting up the URIs if wanted.